AsyncDisplayKit/AsyncDisplayKit.h

```diff
@@ -62,10 +62,11 @@
 #import <AsyncDisplayKit/ASRangeHandler.h>
 #import <AsyncDisplayKit/ASRangeHandlerPreload.h>
 #import <AsyncDisplayKit/ASRangeHandlerRender.h>
-#import <AsyncDisplayKit/ASTextNodeCoreTextAdditions.h>
-#import <AsyncDisplayKit/ASTextNodeRenderer.h>
-#import <AsyncDisplayKit/ASTextNodeShadower.h>
-#import <AsyncDisplayKit/ASTextNodeTextKitHelpers.h>
+#import <AsyncDisplayKit/ASTextKitCoreTextAdditions.h>
+// FIXME: Including this forces the renderer to be compiled as a C header, failing to find c++ std lib.
```

Comment:
Hmm, the renderer is an implementation detail. I wouldn't put it in the public header...